routes/demos.js

```diff
@@ -55,12 +55,19 @@ module.exports = function (express) {
           const sampleUrl = new URL(samplePath, tauExamplesHost);
           // Define number of unnecessary directories to be omitted while downloading.
           const numDirsToCut = sampleUrl.pathname.match(/TAU\/examples\/mobile|wearable\/UIComponents/g) ? 4 : 0;
-          exec(`wget --page-requisites --convert-links --no-host-directories --cut-dirs=${numDirsToCut} --directory-prefix ${projectPath} ${sampleUrl}`, (error, stdout, stderr) => {
+          exec(`wget --spider ${sampleUrl}`, (error, stdout, stderr) => {
             if (error) {
               console.log(error);
               return callback(`Could not download sample from ${sampleUrl}`);
             }
-            callback(null, project, projectPath);
+            else {
+              exec(`wget --page-requisites --convert-links --no-host-directories --cut-dirs=${numDirsToCut} --directory-prefix ${projectPath} ${sampleUrl}`, (error, stdout, stderr) => {
+                if (error) {
+                  console.log(error);
```

Comment:
Please add a comment why we don't return.

Reply:
done